helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -988,6 +997,10 @@ gateway:
   terminationGracePeriodSeconds: 600
   priorityClassName: ""
 
+  # Container configurations
+  container:
```

Comment:
Same as above, No need to add container-level securityContext.

Reply:
I noticed that some things (I think it was readOnlyRootFilesystem and allowPrivilegeEscalation) can't be set at the pod level

Reply:
No need to set `readOnlyRootFilesystem` as the cluster receiver and the gateway doesn't need to access the root directory.

Reply:
Alternatively, you can use this securityContext at container level instead of pod level. I don't think we should maintain 2 different security context for same thing.

Reply:
Ok, I deleted container securityContext configuration and left only the original pod securityContext configuration.